aeron-cluster/src/main/java/io/aeron/cluster/ConsensusControlState.java

```diff
@@ -16,81 +16,24 @@
 package io.aeron.cluster;
 
 import io.aeron.ExclusivePublication;
-import io.aeron.Subscription;
 
 /**
  * The state needed to allow control of the consensus module.
  * <p>
  * This is a record object being passed to external entities.
  *
+ * @param logPublication        publication or null.
+ * @param logRecordingId        log recording id.
+ * @param leadershipTermId      leadership term id.
  * @see ConsensusModuleExtension
  */
```

Comment:
If this is part of the public API, could this not lead to binary incompatibility? 

So a user is using this class and compiled against an older version of Aeron (with a class based ConsensusControlState) and later runs with a newer version of Aeron (with a record based ConsensusControlState).